contracts/utils/Property.sol

```diff
@@ -12,8 +12,10 @@ import "../interfaces/IProperty.sol";
 // common custom errors
```

Comment:
The `utils` folder should be reserved for utilities. The `Property` contract is our main contract. It should be in a different folder.

Reply:
I will resolve this on new PR for refactor